tests/components/wemo/test_init.py

```diff
@@ -92,6 +98,24 @@ async def test_static_config_without_port(hass: HomeAssistant, pywemo_device) ->
     assert len(entity_entries) == 1
 
 
+async def test_reload_config_entry(hass: HomeAssistant, pywemo_device) -> None:
```

Comment:
Given the complicated reload process, should this test maybe be expanded to show that the wemo config entry not only does not error when reloaded but that it also works after a reload?

Reply:
Good idea. Done in [a10d9b](https://github.com/home-assistant/core/pull/96620/commits/4a10d9bf99b3a11277ee95248f9d7983169b3375)